cli/connector/integration_test.go

```diff
@@ -328,25 +328,28 @@ func runTestMain(m *testing.M) int {
 		WorkDir:      workDir,
 		User:         opts.User,
 		Pass:         opts.Pass,
-		WaitStart:    100 * time.Millisecond,
-		ConnectRetry: 3,
+		WaitStart:    time.Second,
```

Comment:
I'm not sure about a seconds timeout between retries. It seems too big for unit tests (but ok for integration). Can we make it smaller (200 ms maybe) and increase retries count?

Reply:
I have reduced `RetryTimeout` to 200 ms.

Reply:
As I see `WaitStart` is waited only one time. And `RetryTimeout` is used after each failure. 
https://github.com/tarantool/go-tarantool/blob/master/test_helpers/main.go#L240-L261